posix.mak

```diff
@@ -244,26 +247,25 @@ dlangspec.verbatim.txt : macros.ddoc verbatim.ddoc dlangspec-consolidated.d
 # Git clone rules
 ################################################################################
 
-# LATEST
-../%-${LATEST}/.cloned :
-	[ -d $(@D) ] || git clone ${GIT_HOME}/$* $(@D)/
-	if [ -d $(@D)/.git ]; then cd $(@D) && git checkout v${LATEST}; fi
+# dmd, druntime
+../%/.clone :
+	[ -d $(@D) ] || git clone --depth=1 ${GIT_HOME}/$* $(@D)/
 	touch $@
 
-# HEAD
-../%/.cloned :
-	[ -d $(@D) ] || git clone ${GIT_HOME}/$* $(@D)/
+# dmd-2.066.1, dub-0.9.21
+../%/.clonev :
```

Comment:
> unexplained

The comment says `dmd-2.066.1, dub-0.9.21`.

> How can we make this even more complicated?

Well it's make and I tried to write a generic rule.
I solved this much simpler now.